service/github/check.go

```diff
@@ -74,20 +65,19 @@ func NewGitHubCheck(cli *github.Client, owner, repo string, pr int, sha, level,
 		SHA:      sha,
 		Level:    level,
 		ToolName: toolName,
-		wd:       workDir,
 	}, nil
 }
 
 // Post posts a reviewdog comment.
 func (ch *Check) Post(_ context.Context, c *reviewdog.Comment) error {
-	c.Result.Diagnostic.GetLocation().Path = filepath.ToSlash(filepath.Join(ch.wd,
-		c.Result.Diagnostic.GetLocation().GetPath()))
 	ch.muComments.Lock()
 	defer ch.muComments.Unlock()
 	ch.postComments = append(ch.postComments, c)
 	return nil
 }
 
+func (*Check) ShouldPrependGitRelDir() bool { return true }
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported method Check.ShouldPrependGitRelDir should have comment or be unexported
<!-- __reviewdog__:ChBjNjY1MTUyODUwZGIyMWRjEg9nb2xpbnQtY2lyY2xlY2k= -->